google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -199,7 +199,7 @@ public void run() {
             }
           });
 
-          LOG.error("Error listing debuggees from Cloud Debugger API", ex);
+          LOG.warn("Error listing debuggees from Cloud Debugger API", ex);
```

Comment:
The user sees the message to select another project with debuggable module in the dialog.